ansible/roles/passenger/files/etc/nginx/nginx.conf

```diff
@@ -51,59 +48,26 @@ http {
   gzip on;
   gzip_disable "msie6";
 
-  # gzip_vary on;
-  # gzip_proxied any;
-  # gzip_comp_level 6;
-  # gzip_buffers 16 8k;
-  # gzip_http_version 1.1;
-  # gzip_types text/plain text/css application/json application/javascript text/xml application/xml application/xml+rss text/javascript;
-
   ##
   # Phusion Passenger config
   ##
-  # Uncomment it if you installed passenger or passenger-enterprise
-  ##
 
   passenger_root /usr/lib/ruby/vendor_ruby/phusion_passenger/locations.ini;
   passenger_default_user deploy;
   passenger_default_group deploy;
   passenger_ruby /home/deploy/.rbenv/shims/ruby;
-
   passenger_max_preloader_idle_time 0;
-  passenger_max_pool_size 10;
-  passenger_min_instances 10;
-  passenger_pool_idle_time 300;
-
+  # Set both the passenger_max_pool_size and the passenger_min_instances as the same count to minimize process spawning
+  # ref: https://www.phusionpassenger.com/library/config/nginx/optimization/#minimizing-process-spawning
+  passenger_max_pool_size 8;
```

Comment:
Changing this from 10 to 8. To optimize this better for CPU-bound requests rather than IO-bound ones. The number of vCPUs on production boxes are now 4 per instance, so this is simply a x2 multiplier. We should hopefully see better utilization of resources with this.

ref: https://www.phusionpassenger.com/library/config/nginx/optimization